awx/network_ui/urls.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2017 Red Hat, Inc
+from django.conf.urls import url
+
+from awx.network_ui import views
+
+app_name = 'network_ui'
+urlpatterns = [
+    url(r'^topology.json$', views.json_topology_data, name='json_topology_data'),
```

Comment:
Tower jobs stdout output is influenced by a GET param `?format=json/html`. I'd rather see 1 url to make future support of the interface easier.

Reply:
Does Tower support YAML export?